code/datums/outfits/outfit.dm

```diff
@@ -222,14 +222,19 @@
 	if(pda && !visualsOnly)
 		var/obj/item/I = new pda(H)
 		switch(H.pda_choice)
+			I.desc_fluff = "For its many years of service, this model has held a virtual monopoly on the PDA market. The secret? A lapel pin affixed to the back."
```

Comment:
this needs to go outside the switch or in its own case, right now this doesn't compile